.github/workflows/ci.yml

```diff
@@ -34,10 +34,11 @@ jobs:
         key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
         restore-keys: |
           ${{ runner.os }}-yarn-
+    - run: cp .env.example .env
     - run: yarn --frozen-lockfile
     - run: yarn install-peers -f
     - run: yarn lint
     - run: yarn build
-    - run: yarn test --coverage
+    - run: yarn test
```

Comment:
why is that?

Reply:
This is not done via jest, so it has no coverage.

Reply:
This is done by default via nyc command (instambul)